src/main/java/io/quarkus/search/app/entity/Guide.java

```diff
@@ -45,27 +50,54 @@ public class Guide {
     @KeywordField
     public String origin;
 
-    @FullTextField(highlightable = Highlightable.UNIFIED, termVector = TermVector.WITH_POSITIONS_OFFSETS)
-    @FullTextField(name = "title_autocomplete", analyzer = AnalysisConfigurer.AUTOCOMPLETE, searchAnalyzer = AnalysisConfigurer.DEFAULT)
-    @KeywordField(name = "title_sort", normalizer = AnalysisConfigurer.SORT, searchable = Searchable.NO, sortable = Sortable.YES)
-    @Column(length = Length.LONG)
-    public String title;
-
-    @FullTextField(highlightable = Highlightable.UNIFIED, termVector = TermVector.WITH_POSITIONS_OFFSETS)
-    @FullTextField(name = "summary_autocomplete", analyzer = AnalysisConfigurer.AUTOCOMPLETE, searchAnalyzer = AnalysisConfigurer.DEFAULT)
-    @Column(length = Length.LONG32)
-    public String summary;
+    @I18nFullTextField(highlightable = Highlightable.UNIFIED, termVector = TermVector.WITH_POSITIONS_OFFSETS, localization = {
+            @Localization(language = "en"),
+            @Localization(language = "es"),
+            @Localization(language = "ja")
```

Comment:
I don't think the default analyzer will make sense for non-english languages.

Same for other fields below.